tests/testthat/test-lazy.R

```diff
@@ -247,6 +247,21 @@ test_that("unique", {
   expect_equal(z, 5)
 })
 
+test_that("unique, maintain_order", {
+  df = pl$DataFrame(
+    x = rep(c("A", "B", "C"), 2),
+    y = rep(c("A", "B", "C"), 2)
+  )
+  # with maintain_order = FALSE, the order could still be maintained once in
+  # a while due to chance so we repeat this test several times
+  for (i in 1:5) {
+    expect_equal(
+      df$lazy()$unique(maintain_order = TRUE)$collect()$to_data_frame()$x,
+      c("A", "B", "C")
+    )
+  }
```

Comment:
I don't see the validity of repeating 5 times here.
Is it possible to reduce the chance factor by fixing seed or setting some environment variable?

Reply:
I don't know how to do that, @sorhawell do you have some idea?

Reply:
We could test on  a column of e.g. a hundred unique elements, then the risk if randomly getting the order right by chance would be astronomically low `1 / 100! = 1E-157`.

Reply:
indeed, that was a very easy solution :smile: